packages/web/src/pages/settings-page/components/desktop/DeveloperApps/DeleteAppConfirmationPage.tsx

```diff
@@ -79,26 +77,23 @@ export const DeleteAppConfirmationPage = (
     <div>
       <h4 className={styles.header}>{name}</h4>
       <p className={styles.confirmation}>{messages.confirmation}</p>
-      <ModalFooter className={styles.deleteAppFooter}>
+      <ModalFooter css={{ paddingBottom: `0 !important` }}>
```

Comment:
😢

Reply:
for your context its cause modal-footer uses css-module. but when we migrate it, this can go away